packages/lesswrong/components/posts/dialogues/DialogueEditorFeedback.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { Dispatch, SetStateAction, useState } from 'react';
+import { getSiteUrl, registerComponent } from '../../../lib/vulcan-lib';
+import Button from '@material-ui/core/Button';
+import { useCurrentUser } from '../../common/withUser';
+
+const styles = (theme: ThemeType): JssStyles => ({
+  root: {
+    padding: 20
+  },
+  button: {
+    fontSize: 16,
+    textTransform: 'none',
+    backgroundColor: theme.palette.buttons.startReadingButtonBackground,
+  },
+  feedbackRow: {
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'center',
+    justifyContent: 'center',
+  }
+});
+
+type clickState = 'unclicked' | 'clickedAndItWorked' | 'clickedAndItDidntWork'
```

Comment:
Types should be PascalCase.  Also probably easier to read something like `'unclicked' | 'success' | 'failure'`, idk